### PR TITLE
Update coding standards in Participated plug-in

### PR DESCRIPTION
**Pull request by initvector**
This update is a coding standards overhaul in the Participated plug-in.

### CODE REVIEW

File: plugins/Participated/class.participated.plugin.php [hunk 1]
```diff
@@ -1,202 +1,247 @@
...
+                $this->countParticipated = false;
+            }
+        }
+        return $this->countParticipated;
+    }
+
+    /**
+     * Gets list of discussions user has commented on.
+     *
+     * @param array $args
```
Comment (DaazKu) [inline plugins/Participated/class.participated.plugin.php]: `$args` does not exist

File: plugins/Participated/class.participated.plugin.php [hunk 2]
```diff
@@ -1,202 +1,247 @@
...
+     * @param mixed $sender
+     */
+    public function base_afterDiscussionFilters_handler($sender) {
+        if (!Gdn::session()->checkPermission('Garden.SignIn.Allow')) {
+            return;
+        }
+
+        // Participated
+        $cssClass = 'Participated';
+        $controller = strtolower(Gdn::Controller()->ControllerName);
```
Comment (DaazKu) [inline plugins/Participated/class.participated.plugin.php]: Casening! You can use this: https://gist.github.com/DaazKu/fc20843e3f7bd2e0311194c0e70dada5

**Comment by initvector** [comment]
fix'd

**Comment by DaazKu** [comment]
http://static2.fjcdn.com/comments/I+approve+this+post+_b46366905f809d2b6085eef5060bb957.jpg

**Pull request merged**